src/PowerShellEditorServices/Services/PowerShellContext/PowerShellContextService.cs

```diff
@@ -639,267 +639,262 @@ public Task<RunspaceHandle> GetRunspaceHandleAsync(CancellationToken cancellatio
                         errorMessages,
                         executionOptions)).ConfigureAwait(false);
             }
-            else
+
```

Comment:
If you hide whitespace changes here, you'll see I just got rid of the `else`. Been wanting to do that for a while.